### ISSUE TITLE
Cannot read pump.io outbox

### ISSUE THREAD

**Issue by stephensekula** [stephensekula/navierstokes#14]
File this under "not sure if it's my instance or PyPump." Here are the symptoms:

* Authenticate a PyPump app (e.g. for NavierStokes) -> this succeeds and generates the necessary credentials, saved to ~/.config/PyPump/credentials.json
* Try to start a pypump shell or just try to create a PyPump object in Python code. Same problems results: I get a "401" error and am told I am not authenticated. Related symptom: try to use the Person object with the PyPump object to retrieve details about my own account, and I get nothing back. The Person object is not filled out, and this appears to be because there is some kind of read-level authentication problem.
* However, I am able to post to pump.io using the same PyPump object! So authentication is working ... sort of ... and inconsistently.

I have started diving into the PyPump code, but no clear discoveries so far. I also have not found any of the old Pump.io sites alive where I could make an account and try this out.

I'll say it out loud in this issue: I think Pump.io is dead. The ecosystem has decayed. I am not sure it's worth the effort to keep supporting this.

Of course, it could just be my pump.io instance configuration, though that has NOT changed in years. I have kept up-to-date with pump.io releases, but there has not been one in years. PyPump is even older in terms of releases and support.

**Comment by jfrobbins** [comment]
> I'll say it out loud in this issue: I think Pump.io is dead. The ecosystem has decayed. I am not sure it's worth the effort to keep supporting this.

😢

I was trying to find some pump.io servers a while back to try to get back into the community, but didn't have much luck. I did fire up the old mastadon,  though.

**Comment by stephensekula** [comment]
Yeah ... you're the voice in my head. I have been holding on to it because I prefer to leave up services until they don't even run any more ... but this one has become so frayed that it's wasting my own time trying to just keep bridging alive. For now, in my own configuration, I have turned off bridging to/from Pump.io.

**Comment by stephensekula** [comment]
Oh, and I too have spun up a Mastodon instance and have an account on there.
